odoo/addons/base/models/assetsbundle.py

```diff
@@ -50,6 +50,10 @@ class AssetNotFound(AssetError):
     pass
 
 
+def _xml_error_message(path, msg):
```

Comment:
I'm not a big fan of managing a specific error here on the root. This versions work for me but here are some suggestion to explore, maybe talk about latter. Im not sure it is applicable but I thought it could be a good idea to manage the error on the WebAsset level to allow specific error management. The rough idea is to have something like:

```python
class WebAsset():
    def generate_error(msg):
        msg = f'{msg} in file {self.url}
        _logger.error(msg)  # log it at least in the python console in all cases. 
        return msg
```

Then, for `XMLAsset`

```python
class XMLAsset():
    def generate_error(msg):
        msg = super().generate_error(msg)
        #TODO  xml escape msg 
        return f'<t t-name="parsing_error{path.replace("/","_")}"><parsererror>{msg}</parsererror></t>'
```

The same idea can be applied to JavascriptAsset

```python
class JavascriptAsset():
    def generate_error(msg):
        msg = super().generate_error(msg)
        # TODO js escape msg
        return f'js_compile_error={msg};console.error({msg})'
```

And StylesheetAsset

```python
class StylesheetAsset():
    def generate_error(msg):
        msg = super().generate_error(msg)
        # TODO css escape msg
        return """
body::before {
  font-weight: bold;
  content: "A css error occured, using an old style to render this page";
  position: fixed;
  left: 0;
  bottom: 0;
  z-index: 100000000000;
  background-color: #C00;
  color: #DDD;
}

css_error_message {
  content: "%s";
}
""" % msg
```

But for StylesheetAsset this may need a little more work and i'm not sure if it is cleanly feasable since the compile is done on all assets But there are already some workaround with `assets[0].compile, source`. Maybe this should be a class method? Also, this will try to get the previous bundle for css and not for other, I don't know if we should do the same or not.

All of that could also be managed in the same place of the scssErrorDisplayService to manage template error the same way.